s2/library/test/s2_test.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const assert = require('assert').strict;
+/* Emulate how BigQuery would load the file */
+global.eval(fs.readFileSync('../../s2_library.js') + '');
+
+describe('S2 unit tests', () => {
+    it('Version', async() => {
+        assert.equal(s2Version(), 1);
+    });
+
+    it('Quadkey/S2 id Conversions', async() => {
+        let z, lat, lng;
```

Comment:
Same as in other cases, I think it would be great to have tests where you check specific values and match to the known good value, so not everything checks that a -> b -> a works.